packages/scripts/src/send-email-blast.js

```diff
@@ -0,0 +1,190 @@
+import fs from 'fs';
+import dotenv from 'dotenv';
+import { readFileSync } from 'fs';
+import aws from 'aws-sdk';
+import * as handlebars from 'handlebars';
+
+import { subjectOne, templateOne } from './constants.js';
+dotenv.config({ path: '../../.env' });
+
+const MAX_RETRY = 5;
+let retryCount = 0;
+let errors = [];
+let sent = 0;
+
+const credentials = new aws.Credentials({
+  accessKeyId: process.env.AWS_ACCESS_KEY_ID,
+  secretAccessKey: process.env.AWS_SECRET_ACCESS_KEY,
+  sessionToken: process.env.AWS_SESSION_TOKEN,
+});
+aws.config.credentials = credentials;
+aws.config.region = process.env.AWS_S3_REGION;
+
+// authenticate aws ses
+const ses = new aws.SES();
+
+// Five requests per second
+const targetRate = 10;
+
+// genereates csv file for any email NOT sent
+function generateEmailCSV(array, outputFilePath) {
+  let fulltext = 'name,email\n';
+  array.map(row => {
+    fulltext += `${row.name},${row.email}\n`;
+  });
+  fs.writeFileSync(outputFilePath, fulltext);
+}
+
+// get emails from csv file
+async function getEmails() {
+  // await new Promise(resolve => {
+  //   setTimeout(resolve(), 1000);
+  // });
+  const file = readFileSync('./in/emails.csv').toString();
+  // Split the string into rows, discard the columns row
+  return file
+    .split('\n')
+    .slice(1)
+    .map((rowString, index) => {
+      // Split each row into it's variables
+      let row = rowString.split(',');
+      return {
+        name: row[0].trim(),
+        email: row[1].trim(),
+        confirmation_id: row[2].trim(),
+      };
+    });
+}
+
+function printProgress(progress) {
+  process.stdout.clearLine();
+  process.stdout.cursorTo(0);
+  process.stdout.write(progress);
+}
+
+// remove duplicate email entries
+function findDuplicates(emails) {
+  let unique = [];
+  let result = [];
+  emails.forEach(email => {
+    if (!unique.includes(email.email)) {
+      unique.push(email.email);
+      result.push(email);
+    } else {
+      console.log('Duplicate found', JSON.stringify(email));
+    }
+  });
+  return result;
+}
+
+// get handlebar template
+// function getTemplate() {
+//   const templatePath = path.resolve(`${__dirname}/templates/email-blast-template.hbs`);
+
+//   const templateContent = fs.readFileSync(templatePath, 'utf-8');
+//   const template = handlebars.compile(templateContent, { strict: true });
+//   const body = template(mailable.context);
+//   return body;
+// }
```

Comment:
Nit - Lets remove this unused code.

Reply:
ok , I just wasn't sure if the template file would be a .hbs or not.